torch/distributions/constraint_registry.py

```diff
@@ -212,3 +212,20 @@ def _transform_to_simplex(constraint):
 @transform_to.register(constraints.lower_cholesky)
 def _transform_to_lower_cholesky(constraint):
     return transforms.LowerCholeskyTransform()
+
+
+@biject_to.register(constraints.cat)
```

Comment:
Oh one more thing, you'll want to register bijections and transforms separately:
```py
@biject_to.register(constraints.cat)
def _biject_to_cat(constraint):
    return transforms.CatTransform([biject_to(c) for c in constraint.cseq],
                                   constraint.dim,
                                   constraint.lengths)

@transform_to.register(constraints.cat)
def _transform_to_cat(constraint):
    return transforms.CatTransform([transform_to(c) for c in constraint.cseq],
                                   constraint.dim,
                                   constraint.lengths)
```
and similarly for `StackTransform`